### PR TITLE
Replaced references to North Star with Enterprise Scale

### PR DESCRIPTION
**Pull request by Mahesh-MSFT**
**This PR fixes**

### CODE REVIEW

File: docs/reference/contoso/armTemplates/auxiliary/policies.json [hunk 1]
```diff
@@ -10727,7 +10727,7 @@
             {
               "Name": "Deploy-Sql-Security",
               "Properties": {
-                "Description": "Recommended built-in security policies for the North Star architecture",
+                "Description": "Recommended built-in security policies for the Enterprise Scale Landing Zone architecture",
```
[the remark this answers is not in the document]
Comment (Mahesh-MSFT) [reply docs/reference/contoso/armTemplates/auxiliary/policies.json:10730]: i do not see any reference to `Deploy-Sql-Security` policy in AdventureWorks and Wingtip reference implementations.

**Review by hjscherer — COMMENTED** [review]
We updated the RI with other changes. @Mahesh-MSFT should we close this PR or do you want to resolve conflicts? If you close it please stage a new PR.

**Comment by hjscherer** [comment]
@Mahesh-MSFT can you resolve conflicts? and replicate changes across all reference implementation. We were driving consistency across for the PolicyDefinitions.

**Comment by Mahesh-MSFT** [comment]
OK to close the PR. have resolved the conflicts anyway.

**Comment by hjscherer** [comment]
Thank you

**Pull request closed without merging**